### PR TITLE
chore(cpp): update subproject dependencies

### PR DESCRIPTION
**Pull request by karjonas**

### CODE REVIEW

File: .gitsubprojects [hunk 1]
```diff
@@ -1,18 +1,18 @@
 # -*- mode: cmake -*-
 if(BRAYNS_NETWORKING_ENABLED)
-  git_subproject(Rockets https://github.com/BlueBrain/Rockets.git cc53765)
+  git_subproject(Rockets https://github.com/BlueBrain/Rockets.git 25c65c3)
 endif()
 
 # Streaming to display walls
 if(BRAYNS_DEFLECT_ENABLED)
-  git_subproject(Deflect https://github.com/BlueBrain/Deflect.git 427458e)
+  git_subproject(Deflect https://github.com/BlueBrain/Deflect.git 761f7d0)
```
Comment (hernando) [inline .gitsubprojects]: This is already outdated

File: .gitsubprojects [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
 # Data access
 if(BRAYNS_CIRCUITVIEWER_ENABLED)
-  git_subproject(Brion https://github.com/BlueBrain/Brion.git d8e7c2e)
+  git_subproject(Brion https://github.com/BlueBrain/Brion.git e581731)
 endif()
 
 if(BRAYNS_BBIC_ENABLED)
-  git_subproject(HighFive https://github.com/BlueBrain/HighFive.git 8db7f3c)
+  git_subproject(HighFive https://github.com/BlueBrain/HighFive.git cd1470a)
```
Comment (hernando) [inline .gitsubprojects]: This is quite old as well.

File: .gitsubprojects [hunk 3]
```diff
@@ -1,18 +1,18 @@
 # -*- mode: cmake -*-
 if(BRAYNS_NETWORKING_ENABLED)
-  git_subproject(Rockets https://github.com/BlueBrain/Rockets.git cc53765)
+  git_subproject(Rockets https://github.com/BlueBrain/Rockets.git 25c65c3)
```
Comment (hernando) [inline .gitsubprojects]: And this one is also old.

**Comment by karjonas** [comment]
I took the latest releases. Do you want to use the development branches?

**Comment by hernando** [comment]
Yes, this is development after all.

**Comment by karjonas** [comment]
done

**Pull request merged**